playbooks/linode-headers.yml

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
You should name this 'linux-headers' instead, because it might be applicable on more than just linode.